docs/preview.rst

```diff
@@ -0,0 +1,7 @@
+Tech previews
+=============
+
+The following features are currently being released as part of a tech preview.
+
+* Distribution (Kickstart) Trees support
```

Comment:
can you elaborate on what kind of support? i suggest specifically specifying that you can sync kickstarts. We do not support upload for example.